google/resource_bigquery_table_test.go

```diff
@@ -64,11 +85,40 @@ func testAccBigQueryTableExists(n string) resource.TestCheckFunc {
 			return fmt.Errorf("No ID is set")
 		}
 		config := testAccProvider.Meta().(*Config)
-		_, err := config.clientBigQuery.Tables.Get(config.Project, rs.Primary.Attributes["dataset_id"], rs.Primary.Attributes["name"]).Do()
+		table, err := config.clientBigQuery.Tables.Get(config.Project, rs.Primary.Attributes["dataset_id"], rs.Primary.Attributes["table_id"]).Do()
```

Comment:
Good catch, thanks! Want to fix that up in `testAccCheckBigQueryTableDestroy` too?